esx_mechanicjob.sql

```diff
@@ -1,30 +1,21 @@
 USE `es_extended`;
 
 INSERT INTO `addon_account` (name, label, shared) VALUES
-	('society_mechanic', 'Mécano', 1)
+	('society_mechanic', 'Mechanic', 1)
 ;
 
 INSERT INTO `addon_inventory` (name, label, shared) VALUES
-	('society_mechanic', 'Mécano', 1)
+	('society_mechanic', 'Mechanic', 1)
 ;
 
 INSERT INTO `jobs` (name, label) VALUES
-	('mechanic', 'Mécano')
+	('mechanic', 'Mechanic')
 ;
 
 INSERT INTO `job_grades` (job_name, grade, name, label, salary, skin_male, skin_female) VALUES
-	('mechanic',0,'recrue','Recrue',12,'{}','{}'),
-	('mechanic',1,'novice','Novice',24,'{}','{}'),
-	('mechanic',2,'experimente','Experimente',36,'{}','{}'),
-	('mechanic',3,'chief',"Chef d\'équipe",48,'{}','{}'),
-	('mechanic',4,'boss','Patron',0,'{}','{}')
-;
-
-INSERT INTO `items` (name, label, weight) VALUES
-	('gazbottle', 'bouteille de gaz', 2),
-	('fixtool', 'outils réparation', 2),
-	('carotool', 'outils carosserie', 2),
-	('blowpipe', 'Chalumeaux', 2),
-	('fixkit', 'Kit réparation', 3),
-	('carokit', 'Kit carosserie', 3)
-;
+	('mechanic',0,'train','Trainee',500,'{"tshirt_2":0,"tshirt_1":15,"shoes_1":63,"torso_2":0,"pants_1":10,"sex":0,"decals_2":0,"decals_1":45,"torso_1":250,"pants_2":0,"shoes_2":2}','{"tshirt_2":0,"tshirt_1":15,"shoes_1":37,"torso_2":0,"pants_1":27,"sex":0,"decals_2":0,"decals_1":53,"torso_1":258,"pants_2":0,"shoes_2":3}'),
```

Comment:
Please don't change the SQL statement, this could break the whole job function.

Reply:
Restored original SQL file, only modified rank names, translated "Mecano" to "Mechanic".

Reply:
job label is already translated into English in the database. SQL is not important because it's already deployed in the system.

Reply:
The English translation is already inside the localization folde.

Reply:
Ohh okay okay po. Restored original sql file po.